src/compilation/build.sh

```diff
@@ -3,6 +3,7 @@
 # Include utils library
 script_dir=$(dirname "$0")
 source "$script_dir/utils.sh"
+source "$script_dir/full_build_conf.sh"
```

Comment:
Make this an optional environment variable.
```suggestion
source "${GDB_STATIC_BUILD_CONF_FILE:-$script_dir/full_build_conf.sh}"
```

Reply:
Not doing this right now, since in order for this to work, we need to pass the env variable into the docker, and our build system is cluttered as is. we can support it when we re-write the build system later on :)